docs/supported-components.human

```diff
@@ -239,3 +239,83 @@ winston
   3.3.1-3.3.3
 zlib
   v14.17.5
+============================================================
+node version 16
+============================================================
+@hapi/hapi
+  20.1.2-20.1.5
+@hapi/vision
+  5.5.0-6.1.0
+amqplib
+  0.2.1-0.3.2
+  0.4.1-0.5.3
+  0.5.5-0.8.0
+bcrypt
+  3.0.6-5.0.1
+bluebird
+  2.0.2-3.7.2
+bunyan
+  1.0.0-2.0.5
+cassandra-driver
+  3.3.0-4.6.3
+co-render
+  0.0.1-1.1.0
+crypto
+  v16.7.0
+director
+  1.2.0-1.2.8
+express
+  3.0.0-4.17.1
+fs
+  v16.7.0
+generic-pool
+  2.4.0-3.8.2
+hapi
```

Comment:
This is not true. 

Since we "skip" test by exiting if node version is low, there is no error, and since there is no error, `testeachversion` considers the version valid. Some tweaks to `versions.js` are required.